extension-compose/src/main/java/com/mapbox/maps/extension/compose/MapboxMap.kt

```diff
@@ -10,6 +9,52 @@ import com.mapbox.maps.MapInitOptions
 import com.mapbox.maps.MapView
 import com.mapbox.maps.MapboxExperimental
 import com.mapbox.maps.MapboxMap
+import com.mapbox.maps.extension.compose.internal.MapApplier
+import com.mapbox.maps.extension.compose.internal.MapNode
+import com.mapbox.maps.extension.compose.internal.MapPreviewPlaceHolder
+import com.mapbox.maps.plugin.gestures.OnMapClickListener
+import com.mapbox.maps.plugin.gestures.OnMapLongClickListener
+import com.mapbox.maps.plugin.gestures.gestures
+import kotlinx.coroutines.awaitCancellation
+
+/**
+ * MapboxMapNode to observe/update the input arguments of MapboxMap.
+ */
+private class MapboxMapNode(
+  val controller: MapView,
+  var clickListener: OnMapClickListener?,
```

Comment:
```suggestion
  internal var clickListener: OnMapClickListener? = {},
```

Reply:
same here, as this is a private class, we don't need to add internal, and I intentionally didn't add default value, as we specifically pass in the click listener during construction.

Reply:
and what about default listener?

Reply:
by default there shouldn't be a listener with nothing done on a click event. Also it will introduce some overhead to pass through the event. It's changed to null by default with discussions in [this](https://github.com/mapbox/mapbox-maps-android/pull/1983#discussion_r1093232950) thread